frontend/src/APIClients/queries.ts

```diff
@@ -1,3 +1,115 @@
-// TODO: move query gql strings here
+import { DocumentNode, gql } from "@apollo/client";
 
-export {};
+// Stories
+export const GET_STORY_CONTENTS = (id: number) =>
+  gql`
+      query {
```

Comment:
Some of the queries are named and others are anonymous (sorry, I suspect half of these are me), apollo docs seem to prefer named - can we make this consistent?